web/src/main/java/com/garret/movies/controller/MovieDatabaseController.java

```diff
@@ -31,6 +35,11 @@ public SimpleMoviesResponse findAll(
         return movieService.getAll(pageable);
     }
 
+    @GetMapping("/by")
+    public List<MovieDto> findByRequestParams(@RequestParam Map<String, String> params) {
```

Comment:
@Garret-vin Map is not a good approach. Better use object, because nobody knows what should be in map in this case ))